python-flask-app/requirements.txt

```diff
@@ -0,0 +1,7 @@
+blinker==1.8.2
```

Comment:
what are all these extra dependencies

Reply:
These are dependencies of `flask`. This file is generated by `pip freeze` so all of the libraries installed are list here.